source/MaterialXContrib/MaterialXMaya/Plugin.cpp

```diff
@@ -52,6 +50,62 @@ void Plugin::initialize(const std::string& loadPath)
         fragmentManager->setEffectOutputDirectory(shaderDebugPath.c_str());
         fragmentManager->setIntermediateGraphOutputDirectory(shaderDebugPath.c_str());
     }
+
+    loadLibraries();
+}
+
+void Plugin::loadLibraries()
```

Comment:
This method can be called repeatedly. Worth adding a command to reload the libraries, and maybe even another button to the AE template? All material documents loaded after the library reload would import the up-to-date versions.